src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+/* CONTAINER */
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+/* LOGO */
+
+.home__link {
+  display: flex;
+  align-items: center;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+}
+
+/* LINKS */
+
+.links {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  margin-left: 20px;
+}
+
+.nav__link:first-child {
+  margin-left: 0;
+}
+
+.text {
+  line-height: 60px;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  display: block;
+}
+
+/* APPLE LINK */
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
+
+/* HOVER */
+
+.nav__link:hover a {
+  color: #00acdc;
```

Comment:
This selector `.nav__link:hover a` seems to be incorrect. It implies that there's an `<a>` tag inside `.nav__link` on hover, but from the HTML structure it's not the case. The `<a>` tag itself has the class `.nav__link`. So it should be `.nav__link:hover` instead.